src/types/Commands.ts

```diff
@@ -11,6 +12,7 @@ interface SlashCommand {
 	data: ChatInputApplicationCommandData;
 	permissions?: ApplicationCommandPermissionData[];
 	execute(interaction: CommandInteraction): Promise<void>;
+	autocomplete?(interaction: AutocompleteInteraction): Promise<{ name: string; value: any }[] | undefined>;
```

Comment:
```suggestion
	autocomplete?(interaction: AutocompleteInteraction): Promise<ApplicationCommandOptionChoice[] | unknown>;
```
discord.js has an interface for a choice object so I thought that'd be good to use, also, the return type could be more than just undefined so I think unknown works. 
P.S. I know I didn't add the import for it, but I couldn't figure out how to add the import at the top and add changes at this line